src/foam/nanos/cron/AdvIntervalSchedule.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'AdvIntervalSchedule',
```

Comment:
Please don't use short forms like this. 
And what does 'Advanced' mean?  What is an Advanced schedule, what do we call the next one we create.

Reply:
We can rename the existing interval schedule to TimeIntervalSchedule.
and/or just rename this one. 
```suggestion
  name: 'DateIntervalSchedule',
```
Open to other suggestions.